agent/src/main/java/com/testsigma/agent/mobile/ios/IosDeviceService.java

```diff
@@ -94,16 +100,49 @@ private Device buildDevice(NSDictionary dico) {
     return deviceAttachMessage;
   }
 
+  public List<MobileDevice> simulatorDeviceList() throws AutomatorException, TestsigmaException {
+    log.info("Fetching iOS device list");
+    List<MobileDevice> deviceList = new ArrayList<>();
+    IosDeviceCommandExecutor iosDeviceCommandExecutor = new IosDeviceCommandExecutor();
+    Process p = iosDeviceCommandExecutor.runDeviceCommand(new String[]{"list-targets", "--json"}, false);
```

Comment:
We need to print the command in logs if we are not doing that

Reply:
We are already doing that